testdata/dnn/download_models.py

```diff
@@ -1056,6 +1061,36 @@ def get_confirm_token(response):  # in case of large files
         member='PP_OCRv3_DB_text_det/PP_OCRv3_DB_text_det.onnx',
         sha='f541f0b448561c7ad919ba9fffa72ff105062934',
         filename='onnx/models/PP_OCRv3_DB_text_det.onnx'),
+    Model(
+        name='age-gender-recognition-retail-0013 (FP32)',
+        url='https://storage.openvinotoolkit.org/repositories/open_model_zoo/2021.2/models_bin/3/age-gender-recognition-retail-0013/FP32/age-gender-recognition-retail-0013.xml',
+        sha='9b99a163614a8f59bf90d5b094a98f0d3f63eb3c',
+        filename='omz_intel_models/intel/age-gender-recognition-retail-0013/FP32/age-gender-recognition-retail-0013.xml'),
```

Comment:
OMZ models are downloaded using OMZ downloader.

Reply:
This was a request by @asmorkalov: https://github.com/opencv/ci-gha-workflow/pull/109#issuecomment-1662188849

Reply:
Problem is that path should not be

`<OPENCV_DNN_TEST_DATA_PATH>/dnn/omz_intel_models/...`

Correct path looks like:

`<OPENCV_DNN_TEST_DATA_PATH>/omz_intel_models/...`

See `initDLDTDataPath()`.

Reply:
Oh, got it. Removed OpenVINO models.